lib/internal/test_runner/test.js

```diff
@@ -746,6 +769,8 @@ class TestHook extends Test {
 }
 
 class Suite extends Test {
+  omitFromCounters = true;
+  reportedType = 'suite';
```

Comment:
Maybe we should have a `type` field on `Test` and everything that extends from it. We could differentiate between tests, suites, hooks, etc.

We could also use `Symbol.toStringTag` instead of a normal property or `instanceof` (although that one is probably a bigger perf hit).

Reply:
isnt this what I did? not what else are you suggesting?

Reply:
I was suggesting adding it to `Test` and all of the classes that extend `Test`. Right now it's only on `Suite`.